### DISCUSSION TITLE
What about having a banner?

### DISCUSSION THREAD

**Discussion by jamesgeorge007** [verdaccio/verdaccio#1397] [Ideas]
Output of `verdaccio --info`

![Screenshot from 2019-07-19 23-45-02](https://user-images.githubusercontent.com/25279263/61556449-714dbf80-aa7f-11e9-8b7b-dc44612a2886.png)

**Comment by jamesgeorge007** [comment]
I have put together a package for the purpose: [node-banner](npmjs.com/package/node-banner)

**Comment by juanpicado** [comment]
cc: @verdaccio/collaborators @verdaccio/core-team 

If you link https://opencollective.com/verdaccio at the end with a nice message saying encouraging people to donate or contribute, I'm in.  But look pretty nice. Let's see what others think.

**Comment by juanpicado** [comment]
btw, what did you use to achieve that ? :-)

**Comment by DanielRuf** [comment]
I think there are already some banner plugins.

I'm not a big fan of such FIGlet solutions in general. Generally we can hardcode it using a template string + kleur (or one of the FIGlet implementations).

**Comment by DanielRuf** [comment]
We already use kleur and adding the (optional) banner can be easily achieved. I don't think that we need another package which may reference a different kleur version.

Let's reuse and directly integrate it.

@juanpicado figlet and kleur (see the dependencies tab of https://npmjs.com/node-banner).

**Comment by juanpicado** [comment]
I think @DanielRuf is right. Less dependencies, less headache.

**Comment by jamesgeorge007** [comment]
Since this is irrelevant as per now shall we close this :thinking:

**Comment by juanpicado** [comment]
I think you misunderstood the feedback, it's fine, but should be "home made" rather using a dependency. Using a template or something is an option, but within our source code, easy to change or update, does it make sense for you?

**Comment by juanpicado** [comment]
Actually, I'd reuse the header. Something like to have a future `verdaccio --about` command in the future, displaying key info for the project, url, chat, donations and some relevant data.

**Comment by juanpicado** [comment]
@jamesgeorge007 are you still on this?

**Comment by jamesgeorge007** [comment]
I'm out of town, will shoot up a PR soon

**Comment by jamesgeorge007** [comment]
I'm not that sure on how to give it a kick-start (as what you guys intend).
@juanpicado maybe you could give me a better picture on how it should be :thinking: 
cc @verdaccio/core-team

**Comment by juanpicado** [comment]
@jamesgeorge007 just use imagination, but I'd recommend do not add new dependencies and if you have to, then should be hosted in our *monorepo*.
